ops/cloud-deployment/backend-api-deploy.bicep

```diff
@@ -200,6 +202,16 @@ resource pacerKVManagedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentitie
 var pacerKeyVaultManagedIdentity = pacerKVManagedIdentity.id
 var pacerKeyVaultManagedIdentityClientId = pacerKVManagedIdentity.properties.clientId
 
+module appInsights './app-insights/app-insights.bicep' = if (deployAppInsights) {
+  name: '${functionName}-appi-module'
+  params: {
+    location: location
+    kind: 'web'
+    appInsightsName: 'appi-${functionName}'
```

Comment:
I almost thought this was a typo for `api`. Would it make sense to spell it out a bit more to `app-insights-`?

Reply:
Believe it or not, this follows the naming conventions Microsoft provides for the app insights resource. However, it is not my favorite naming convention

Reply:
yeah that naming convention is not my fave. I say we go with what makes sense for the team.